rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    book.rentals << self
+    @person = person
+    person.rentals << self
+  end
+end
```

Comment:
- [ ] It seems like you forgot to implement the "A rental belongs to a book and also belongs to a person" relationship. Kindly add it.

![image](https://user-images.githubusercontent.com/86318284/195913072-4d4aabbd-5f85-43bf-ba65-61e9855347be.png)

Reply:
Hi @Nemwel-Boniface 
Please refer to my comment on the first change requested in this PR